src/main/java/bio/terra/service/filedata/DataResultModel.java

```diff
@@ -0,0 +1,14 @@
+package bio.terra.service.filedata;
```

Comment:
I think this abstract class should live in the `bio.terra.service.tabulardata` package probably